lib/ret/project.ex

```diff
@@ -116,8 +116,8 @@ defmodule Ret.Project do
       ) do
     project
     |> changeset(account, params)
-    |> put_change(:project_owned_file_id, project_owned_file.owned_file_id)
-    |> put_change(:thumbnail_owned_file_id, thumbnail_owned_file.owned_file_id)
+    |> put_assoc(:project_owned_file_id, project_owned_file)
```

Comment:
This is unrelated but an iteration of this involved creating dummy projects and this was changed. Feels silly to revert it as its an improvement.